modules/dataframe/tests/unittests/csvreader-test.cpp

```diff
@@ -131,7 +131,7 @@ TEST(CSVdata, emptyFields) {
     auto value = dataframe->getColumn(3)->get(1, true)->toString();
     EXPECT_EQ("", value) << "empty field at end of row";
     value = dataframe->getColumn(1)->get(2, false)->toString();
-    EXPECT_EQ("nan", value) << "empty numeric field at begin of row";
+    EXPECT_EQ("0", value) << "empty numeric field at begin of row";
```

Comment:
No, how should we deal with missing numbers?
This will break existing code assumptions. We need a way to deal with missing numbers if this is removed

Reply:
"nan" still holds true for float32/float64 cols! Note that this column is an integer column, hence no `nan`.

Reply:
Missing numbers -> need to re-design the entire DataFrame/Column datastructures to take care of this. One option would be to use `std::optional` for each single value... Might not be what we want. There are also more issues we need to address like the implicit (or rather explicit) index column at index 0. When copying columns that one needs special handling. One might even remove it from a dataframe... Needs to be discussed before changing anything.

Reply:
I guess one option is also to keep references to missing values. Really not optimal but would go in line with the current implementation